src/app/pages/ArticlePage/fixtureData.ts

```diff
@@ -75,6 +76,9 @@ const articleDataBuilder = (
     },
     tags: things,
     allowAdvertising,
+    stats: {
+      readTime: readTime || undefined,
```

Comment:
Now we don't need this, since it will always have a value
```suggestion
      readTime,
```